CONTRIBUTING.md

```diff
@@ -1,80 +1,141 @@
+
+    Q4.1. Merge requests naming
 
-2. Add the local checkout as a local repository in Phoenicis (see [instructions](https://phoenicisorg.github.io/phoenicis/Users/repository/#local-directory))
-
-3. Select the right category for the application
-    * Accessories
-    * Development
-    * Games
-    * Graphics
-    * Internet
-    * Multimedia
-    * Office
-    * Other
-    * Science
-  
-4. Create a new folder for your script inside that category. The folder structure should be:
     ```
-    category
-    └── application-name
-        ├── script-name
-        │   ├── script.js
-        │   └── script.json
-        ├── miniatures
-        │   └── main.png
-        └── application.json
+    <Category>/<Title> : <short_summary>
+
+    <Fixes/Depends/refference> : <HYPERLINK>
+    <Changelog>
+    <Changelog>
+    ...
+    Signed-off-by : <NAME> <SURNAME> <<EMAIL>>
     ```
 
-    Typically, `script-name` will be something like "Steam", "Online" or "v1.2".
-    
-    Even if the application name contains ®, ™ or the like, you should not use it in the folder name.
-
-5. Fill the files:
-    * [script.js](https://phoenicisorg.github.io/scripts/Develop/script-js/): actual installation script
-    * [script.json](https://phoenicisorg.github.io/scripts/Develop/script-json/): describes the installation script
-        ```json
-        {
-		"scriptName": "Online",
-		"compatibleOperatingSystems": ["MACOSX", "LINUX"],
-		"testingOperatingSystems": [],
-		"free": true,
-		"requiresPatch": false
-        }
-        ```
-    * main.png: application icon (400px x 300px)
-    * [application.json](https://phoenicisorg.github.io/scripts/Develop/application-json/): describes the application
-        ```json
-        {
- 		"name":	"Steam",
-		"description": "Application description"
-        }
-        ```
-  
+Q5. All installers has to work as expected to be approved in official and non-testing repository, if the proposed change is not applicable to be accepted in a repository comments in code are accepted otherwise the installers should be comment-free.
+
+Q999. Everyone has to follow set of rules below
+
+#### Code of Contribution
+These set of rules ensures quality of provided contribution making their merge into master more effective and possible without sacrificing the quality of provided code and contribution.
+
+
+C1. **Trolling is not tollerated**
+*Purposefully participation only to aggrevate other parties of phoenicis is not tollerated and will result in a ban for six hours times two if the issue presist after the ban.*
+
+C2. **Spamming is not tollerated**
+*Purposefully posting content that is not relevant to the project and/or propagates NSFW/malicious content without person's consciousness is not tolerated and will result in a ban for 1 month
+
+If person's account was hijacked he/she can contact team in PhoenicisOrg/PublicRelation and requst unban assuming that person did neccesary procautions to prevent such outcome in the future.*
```

Comment:
New concept: PhoenicisOrg/PublicRelations

Recommends making a PublicRelation repository or bugzilla to resolve PR conflicts based on CONTRIBUTING.md